spec/requests/api_controller_spec.rb

```diff
@@ -67,7 +84,7 @@ class Model
     before { model.valid? }
 
     it 'returns the correct errors' do
-      expect(api_controller.send(:validation_errors, model.errors)).to eq errors
+      expect(api_controller.send(:validation_errors, model)).to eq errors
```

Comment:
not sure if `validation_errors` will always return these errors in the prescribed order - maybe use `match_array` rather than `eq`?

Reply:
In the general case, you're right - it's model dependant, but here the errors will be in the order specified in the `Model` class defined within the spec. I think from a documentation point of view it's better to change as suggested to indicate the order is not really defined.

Reply:
👍